our-work/software/omni.yml

```diff
@@ -0,0 +1,20 @@
+title: OpenMind Neuromodulation Interface (OMNI)
+description: OMNI is a software framework for interfacing with neuromodulation systems in a platform, device, and programming language agnostic way.
+languages:
+  - typescript
+  - c#
+  - python
+tags:
+  - neuroscience
+  - neuroengineering
+people:
+  - name: Bradford Roarr
+    github_user: broarr
+groups:
+  - borton-lab
+  - openmind-consortium
+links:
```

Comment:
I think the links for OMNI should be related to OMNI and not Borton's lab or Open-end. Those should be covered by the collaborations cards